cbor/shared/src/test/scala/fs2/data/cbor/ParserSpec.scala

```diff
@@ -394,10 +398,14 @@ object ParserSpec extends SimpleIOSuite {
       .emits(streamingTestCases)
       .evalMap { case (expectedLow, expectedHigh, input) =>
         val bytes = Stream.chunk(Chunk.byteVector(input))
-        (bytes.through(items[IO]).compile.toList.attempt, bytes.through(values[IO]).compile.toList.attempt).mapN {
-          (low, high) =>
-            expect(low == Right(expectedLow), s"CBOR item parser should parse ${input.toHex} properly") and
-              expect(high == Right(expectedHigh), s"CBOR value parser should parse ${input.toHex} properly")
+        (bytes
+           .through(items[IO])
+           .compile
+           .toList
+           .attempt,
+         bytes.through(values[IO]).compile.toList.attempt).mapN { (low, high) =>
+          expect(low == Right(expectedLow), s"CBOR item parser should parse ${input.toHex} properly") and
```

Comment:
For all tests: Shall we switch to `expect.eql/same`? It conveys the intent better and has special rendering in case of mismatches…